e2e-tests/spring-graphql/src/main/kotlin/com/foo/graphql/allLimitReached/DataRepository.kt

```diff
@@ -0,0 +1,52 @@
+package com.foo.graphql.allLimitReached
```

Comment:
should avoid upper-case letter in the name of packages. see:
https://github.com/EMResearch/EvoMaster/blob/master/docs/for_developers.md#naming-convention

Reply:
Done